### DISCUSSION TITLE
Just thank you

### DISCUSSION THREAD

**Discussion by JPFrancoia** [openobserve/openobserve#1712] [General]
Hello,

I just wanted to say thank you, I've been looking for a tool like openobserve for a while.
I host a single-node Kubernetes at home, where I deploy some of my applications. I want a simple tool for observability (tracing and logs mostly). Don't need anything fancy, I just need it to work, to be reliable, and to be simple (I don't want to spend too much time on this).  I used stuff like Grafana + Loki, but god they are hard to configure and install (and I never managed to get the logs compaction working, so my disk usage always exploded).

With Openobserve, it took me literally 10 minutes to get going. Thank you for providing one simple Kubernetes manifest file (and not an opaque helm chart).

Also thank you for making it compatible with opentelemetry. It took me another 10 min to hook my applications.

This tool is perfect for self-hosting. Thank you again.

**Comment by prabhatsharma** [comment]
Thank you for the kind words @JPFrancoia . This keeps us motivated to build. Would love to get any feedback that you might have on the product.
